pkg/preference/machine_output.go

```diff
@@ -60,7 +60,7 @@ func toPreferenceItems(prefInfo preferenceInfo) []api.PreferenceItem {
 		{
 			Name:        ImageRegistrySetting,
 			Value:       settings.ImageRegistry,
-			Default:     "",
+			Default:     DefaultDevfileRegistryURL,
```

Comment:
Default value is intentionally the empty string, which disables the "image name as selector" feature (introduced in #6677).

```suggestion
			Default:     "",
```

Reply:
My bad, you're right :face_exhaling: